Fix `default-branch-button` in "old design"

FloEdelmann opened the pull request:
Follows #3221. Sorry for not testing with "Repository refresh" beta disabled :/

Thanks @yakov116 for catching this in https://github.com/sindresorhus/refined-github/pull/3214#issuecomment-645008022

yakov116 commented on source/features/default-branch-button.tsx:
```diff
@@ -43,7 +44,7 @@ async function init(): Promise<false | void> {
 	);
 
-	if (branchSelector.classList.contains('btn-sm')) {
+	if (select.exists('[data-hotkey="w"].btn-sm', branchSelector)) {
```
Does this have any negative effect since domReady is set false?

FloEdelmann commented:
Thanks again @yakov116, that may have caused a race condition. I've updated the code.

Also, I've fixed the dropdown position:

### Before

![Screenshot_2020-06-17_18-28-20](https://user-images.githubusercontent.com/202916/84924329-a7008b80-b0c8-11ea-9896-fb5c8651f0bd.png)

### After

"Repository refresh" beta enabled:  
![Screenshot_2020-06-17_18-27-56](https://user-images.githubusercontent.com/202916/84924356-af58c680-b0c8-11ea-8888-8c7f2b362571.png)

"Repository refresh" beta disabled:  
![Screenshot_2020-06-17_18-28-45](https://user-images.githubusercontent.com/202916/84924359-aff15d00-b0c8-11ea-9027-99db1270218e.png)

fregante commented:
Thanks @FloEdelmann and @yakov116 

Please include the 3 pieces of information asked in the PR template when opening PRs
